devbox.json

```diff
@@ -20,7 +20,9 @@
     "kyverno-chainsaw@latest"
   ],
   "shell": {
-    "init_hook": [],
+    "init_hook": [
+      "export GOROOT=$(go env GOROOT)"
```

Comment:
nit<sup>2</sup> (syntax):
```suggestion
      "export \"GOROOT=$(go env GOROOT)\""
```
I 🍝'd this from the Devbox Go example 😂: https://github.com/jetpack-io/devbox/blob/main/examples/development/go/hello-world/devbox.json#L7